<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Michael Zietz" />
  <meta name="dcterms.date" content="2018-12-15" />
  <meta name="keywords" content="markdown, publishing, manubot" />
  <title>Network Methods for Multiomic Data Integration</title>
  <style type="text/css">
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
  </style>
  <link rel="stylesheet" href="github-pandoc.css" />
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <!-- Insert Analytics Script Below -->
  <script>
  </script>
  <!-- End Analytics Script -->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Network Methods for Multiomic Data Integration</h1>
</header>
<p><small><em> This manuscript (<a href="https://zietzm.github.io/integration-review/v/e6daa0c5fbced1188c58e1824a4e4a57df1b95d5/">permalink</a>) was automatically generated from <a href="https://github.com/zietzm/integration-review/tree/e6daa0c5fbced1188c58e1824a4e4a57df1b95d5">zietzm/integration-review@e6daa0c</a> on December 15, 2018. </em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><strong>Michael Zietz</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/0000-0003-0539-630X">0000-0003-0539-630X</a> · <img src="images/github.svg" alt="GitHub icon" width="13" height="13" /> <a href="https://github.com/zietzm">zietzm</a><br> <small> Department of Physics, Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania </small></li>
</ul>
<h2 id="abstract" class="page_break_before">Abstract</h2>
<h2 id="introduction">Introduction</h2>
<p>In the distant past, medical knowledge advanced toward the goal of finding treatments for every condition, defined very loosely. Once a treatment was found, the mystery disappeared and the disease had essentially been solved. Massive improvements have been made in the quality of medicine and the quantity of medical knowledge under this paradigm. At the current stage, however, diminishing returns can be achieved by simply trying to find a treatment for a broadly-defined disease. Two major avenues for future improvement stand out. First, the implementation of knowledge in real-world clinical settings. Despite the immensity of hard-gotten medical information available, patients continue to die from curable diseases and medical errors. Without putting into practice known ways to improve medicine, future research will fail to achieve its life-saving potential. Second, medical research must broaden its scope to encompass more information while zooming in to identify the specific biological foundations of disease, including the differences between individuals at the molecular level. Research toward the second goal is known as precision medicine, and this review covers a sub-field within precision medicine research.</p>
<h3 id="omics-and-precision-medicine">Omics and precision medicine</h3>
<p>Precision medicine entails custom-tailoring health care to individual patients based on their personal, clinical, and molecular profiles. The goal of precision medicine is to improve prevention, diagnosis, and treatment by accounting for the medically-relevant differences between individuals and groups. Toward this goal, the clinical value of omics data and omics-based discoveries has been demonstrated repeatedly. A brief overview of two sub-fields illustrates the possibilities for future research and the potential value of these discoveries.</p>
<p>Pharmacogenomics studies the genomic underpinnings of individual differences in drug response. Compared to the earlier-established field of pharmacogenetics, pharmacogenomics focuses on the simultaneous effects of many genes on drug metabolism and toxicity <span class="citation" data-cites="NzgwvuOF">[<a href="#ref-NzgwvuOF">1</a>]</span>. As of August 2018, the FDA lists 284 drugs with pharmacogenomic information included on the label <span class="citation" data-cites="6IpGeQqM">[<a href="#ref-6IpGeQqM">2</a>]</span>.</p>
<p>Deep phenotyping attempts to understand and fully classify the biological components underlying disease <span class="citation" data-cites="16REdtjMj">[<a href="#ref-16REdtjMj">3</a>]</span>. This task is made challenging by the fact that there are often not many well-understood links between genotype and phenotype, especially in the context of complex disease. Attention to varied patient information and its meaningful incorporation will require a new paradigm in medical diagnosis. To better understand diverse patient subtypes, consideration must be given to the patient’s full range of other diagnoses and symptoms, even if seemingly orthogonal <span class="citation" data-cites="gIKXoMmn">[<a href="#ref-gIKXoMmn">4</a>]</span>.</p>
<p>The ideal future for precision medicine would allow patients to be categorized into homogeneous groups for prevention, diagnosis, and treatment. That is, precision medicine seeks to uncover and account for the full range of individual variation that underlies differential disease succeptibility, prognosis, and treatment response. Whether in the form of specific clinical traits or detailed multiomic molecular attributes, precision medicine shows promise to overcome the heterogeneity of patients found in previously loosely-defined disease categories. If disease categorizations truly account for the totality of relevant factors, then patients within homogeneous categories can be treated systematically by the establishment of subtype-specific treatment regimes. Working toward such goals requires a broad approach, in which data from many different sources can be incorporated toward a more complete understanding of disease.</p>
<h3 id="multiomics">Multiomics</h3>
<p>While the clinical application of genomic data shows promise, biology cannot be fully explained by a single data type. Many diseases are affected in part by patients’ genomic profiles, though few diseases are based soley in genetics. The central dogma of molecular biology states that genetic information flows from nucleic acid to nucleic acid and from nucleic acid to protein. This dogma alone suggests that processes and information beyond the genetic sequence of a patient’s DNA will influence phenotype. <em>Multiomics</em> refers to the set of such biologically-relevant molecular data sources: genomics, epigenomics, transcriptomics, proteomics, metabolomics, and the organismal microbiome. Complex diseases may be most explainable when viewed through the lenses of multiple such data sources.</p>
<p>In recent years, the cost of omics data generation has fallen dramatically. A former target of $1,000 per genome sequenced now seems high as companies like Illumina promise a future of the, “$100 genome,” <span class="citation" data-cites="Ml77ulah">[<a href="#ref-Ml77ulah">5</a>]</span>. Such rapid technological improvements have recently made the creation of large-scale multiomic datasets cost-effective. The Cancer Genome Atlas (TCGA) is a large-scale project which has collected patient information and tumor samples from over 11,000 patients in the United States. Its data include clinical information about patients, sample metadata, and multiomic sample data including gene expression, SNP genotypes, copy number variation, DNA methylation, exon sequencing, and microRNA profiles <span class="citation" data-cites="AK17eOgD">[<a href="#ref-AK17eOgD">6</a>]</span>. Another project underway is the International Cancer Genome Consortium (ICGC), which seeks to incorporate data from TCGA and around 50 other projects to provide researchers access to a massive quantity of tumor omics data <span class="citation" data-cites="ynLCqVng">[<a href="#ref-ynLCqVng">7</a>]</span>. The proliferation and falling costs of next-generation sequencing technologies have allowed these and other large consortia to create massive data resources covering many modalities and diverse diseases. Decreasing costs of data have not, however, resulted in a decrease in the cost of data analysis. Significant future research is needed to deeply probe multiomic datasets in order to extract the data that will guide future diagnostics and treatments.</p>
<h3 id="data-integration">Data integration</h3>
<p>Different data modalities can contain distinct pieces of medically-relevant information. This has led researchers and clinicians to consider how multiple data modalities can together provide a more complete picture of disease. An obvious approach for prediction or classification is the simple combination (or concatenation) of data from multiple modalities. The issue with such approaches has to do with the data themselves. Omics data often contain a very large number of features relative to the number of samples. This leads to a low signal-to-noise ratio, which is only worsened by the concatenation of data from two or more different streams. An alternative and promising approach involves extracting information from data modalities individually before combining extracted data into a single, more cohesive representation with a relatively higher signal-to-noise ratio. Several such techniques have been published and a subset of them will be covered later in this review.</p>
<p>Data-integrative methods are advantageous because they allow researchers to discover connections which are attested by multiple sources of data. This advantage is extremely beneficial, as it allows integrative methods to achieve fewer false positives when compared to single-modality methods. Particularly in the age of high-throughput omics technologies, where data can be staggeringly abundant, false positives are a major problem and steps must very often be taken to reduce their influence.</p>
<p>Another advantage of methods which simultaneously consider multiple sources of data is their superior ability to uncover complex and emergent relationships. Data-integrative approaches to multiomic analysis are better suited to capture nonlinear relationships across data modalities than the simple combination of single-modality-derived conclusions. To better illustrate this point, consider the example biological system proposed by Ritchie et al. <span class="citation" data-cites="zR4nSSeT">[<a href="#ref-zR4nSSeT">8</a>]</span>. They imagine two competing hypotheses to explain a complex phenotype. First, so-called “Hypothesis A” is a linear system of explanation in which genomic variation leads to variation in gene expression, leading to variation in protein expression, leading to a certain phenotype. The authors propose that Hypothesis A would be best tested by step-wise progress and incremental data reduction/filtering. An alternative system, “Hypothesis B”, the nonlinear hypothesis, involves multi-interacting connections between genome, transcriptome, proteome, epigenome and phenotype. Hypothesis B, the authors suggest, would be better tested using methods that combine the relevant data modalities first, rather than filtering each type to those entities believed <em>a priori</em> to be relevant to the phenotype.</p>
<p>As an even simpler example, consider the exclusive or (XOR) function. It is impossible to learn the behavior of XOR while only able to modulate or observe one input. Only by observing both relevant inputs can the function be determined. Similarly, to completely understand complex and nonlinear biological systems, hidden variables must be unveiled and understood in the context of all the relevant data. This task is best achieved by simultaneously considering information from multiple sources, the goal of data integration methods.</p>
<h3 id="network-approaches-to-data-integration">Network approaches to data integration</h3>
<p>Networks are a logical way of representing many types of biological data. Nodes (or vertices) may represent biological or biomedical entities, while edges may represent a number of different connections or pairings between entities. Several graph-theoretic layers can be helpful in the construction and analysis of biological networks, including graph coloring, edge weighting, and the specification of edge direction. By including such additional layers of information, as well as the association of nodes and edges with biological metadata, networks can consolidate many types of information into a single, cohesive format.</p>
<p>This review focuses on network-based methods for data integration, though such methods are not the only ones which show promise. For further discussion of alternative approaches, several excellent reviews and comparisons are available <span class="citation" data-cites="zR4nSSeT 156UVA1sT lOKGIFIL">[<a href="#ref-lOKGIFIL">10</a>,<a href="#ref-zR4nSSeT">8</a>,<a href="#ref-156UVA1sT">9</a>]</span>.</p>
<h2 id="discussion">Discussion</h2>
<!-- ### Unsupervised methods -->
<p>Similarity Network Fusion (SNF), a method due to Wang, et al. <span class="citation" data-cites="1Hk7MyXmO">[<a href="#ref-1Hk7MyXmO">11</a>]</span>, uses patient similarity networks to cluster patients and predict future labels. In these networks, nodes represent patients and the edges connecting patients represent their pairwise similarity. For each additional data modality, SNF creates a similarity network based on the patients’ molecular profiles. The constructed modality-specific networks are then fused using an iterative linear algebra routine which is appropriate for studies of varied sample size and feature number, as well as for highly heterogeneous data sources. The authors applied SNF to three data modalities for a case study of glioblastoma multiforme (GBM), an aggressive brain tumor which is challenging to treat <span class="citation" data-cites="vvARGimo">[<a href="#ref-vvARGimo">12</a>]</span>. Previous data-integrative approaches had shown varied results, including the identification of a variable numbers of disease subtypes, depending on the data modality under investigation. Using SNF, the authors were able to recover known clinically-relevant disease subtypes and show that these subtypes correspond to very different survival prognoses. Moreover, by probing the network resulting from the SNF method, the authors were able to show that the majority of the edges were supported by two or more different data modalities.</p>
<!-- ### Supervised methods -->
<p><span class="citation" data-cites="X7vddlEk">[<a href="#ref-X7vddlEk">13</a>]</span> <span class="citation" data-cites="1rJXR1pc">[<a href="#ref-1rJXR1pc">14</a>]</span> <span class="citation" data-cites="18IBB9pE2">[<a href="#ref-18IBB9pE2">15</a>]</span> <span class="citation" data-cites="DyO707G8">[<a href="#ref-DyO707G8">16</a>]</span> <span class="citation" data-cites="glnTeD5X">[<a href="#ref-glnTeD5X">17</a>]</span> <span class="citation" data-cites="8kVDErgV">[<a href="#ref-8kVDErgV">18</a>]</span> <span class="citation" data-cites="14murNoRs">[<a href="#ref-14murNoRs">19</a>]</span> <span class="citation" data-cites="aPrQxP1f">[<a href="#ref-aPrQxP1f">20</a>]</span></p>
<!-- Review -->
<p><span class="citation" data-cites="KOD2gdVS">[<a href="#ref-KOD2gdVS">21</a>]</span></p>
<h2 id="references" class="page_break_before">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs">
<div id="ref-NzgwvuOF">
<p>1. <strong>Pharmacogenomics: Translating Functional Genomics into Rational Therapeutics</strong><br />
W. E. Evans<br />
<em>Science</em> (1999-10-15) <a href="https://doi.org/cw248b">https://doi.org/cw248b</a><br />
DOI: <a href="https://doi.org/10.1126/science.286.5439.487">10.1126/science.286.5439.487</a></p>
</div>
<div id="ref-6IpGeQqM">
<p>2. <a href="https://www.fda.gov/drugs/scienceresearch/ucm572698.htm">https://www.fda.gov/drugs/scienceresearch/ucm572698.htm</a></p>
</div>
<div id="ref-16REdtjMj">
<p>3. <strong>Deep phenotyping for precision medicine</strong><br />
Peter N. Robinson<br />
<em>Human Mutation</em> (2012-04-13) <a href="https://doi.org/gfpptq">https://doi.org/gfpptq</a><br />
DOI: <a href="https://doi.org/10.1002/humu.22080">10.1002/humu.22080</a> · PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22504886">22504886</a></p>
</div>
<div id="ref-gIKXoMmn">
<p>4. <strong>Deep phenotyping: The details of disease</strong><br />
Cathryn M. Delude<br />
<em>Nature</em> (2015-11) <a href="https://doi.org/gfpptr">https://doi.org/gfpptr</a><br />
DOI: <a href="https://doi.org/10.1038/527s14a">10.1038/527s14a</a> · PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/26536218">26536218</a></p>
</div>
<div id="ref-Ml77ulah">
<p>5. <strong>Cheaper DNA sequencing unlocks secrets of rare diseases</strong><br />
Sarah Neville<br />
<em>Financial Times</em> (2018-03-05) <a href="https://ft.com/content/017a3a50-f6f1-11e7-a4c9-bbdefa4f210b">https://ft.com/content/017a3a50-f6f1-11e7-a4c9-bbdefa4f210b</a></p>
</div>
<div id="ref-AK17eOgD">
<p>6. <strong>The Cancer Genome Atlas Pan-Cancer analysis project</strong><br />
John N WeinsteinEric A Collisson, Gordon B Mills, Kenna R Mills Shaw, Brad A Ozenberger, Kyle Ellrott, Ilya Shmulevich, Chris Sander, Joshua M Stuart<br />
<em>Nature Genetics</em> (2013-10) <a href="https://doi.org/f3nt5c">https://doi.org/f3nt5c</a><br />
DOI: <a href="https://doi.org/10.1038/ng.2764">10.1038/ng.2764</a> · PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24071849">24071849</a> · PMCID: <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3919969">PMC3919969</a></p>
</div>
<div id="ref-ynLCqVng">
<p>7. <strong>International network of cancer genome projects</strong><br />
Thomas J. Hudson (Chairperson), Warwick Anderson, Axel Aretz, Anna D. Barker, Cindy Bell, Rosa R. Bernabé, M. K. Bhan, Fabien Calvo, Iiro Eerola, Daniela S. Gerhard, … Huanming Yang<br />
<em>Nature</em> (2010-04-15) <a href="https://doi.org/cm9h2m">https://doi.org/cm9h2m</a><br />
DOI: <a href="https://doi.org/10.1038/nature08987">10.1038/nature08987</a> · PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20393554">20393554</a> · PMCID: <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2902243">PMC2902243</a></p>
</div>
<div id="ref-zR4nSSeT">
<p>8. <strong>Methods of integrating data to uncover genotype–phenotype interactions</strong><br />
Marylyn D. Ritchie, Emily R. Holzinger, Ruowang Li, Sarah A. Pendergrass, Dokyoon Kim<br />
<em>Nature Reviews Genetics</em> (2015-01-13) <a href="https://doi.org/bg9k">https://doi.org/bg9k</a><br />
DOI: <a href="https://doi.org/10.1038/nrg3868">10.1038/nrg3868</a> · PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25582081">25582081</a></p>
</div>
<div id="ref-156UVA1sT">
<p>9. <strong>More Is Better: Recent Progress in Multi-Omics Data Integration Methods</strong><br />
Sijia Huang, Kumardeep Chaudhary, Lana X. Garmire<br />
<em>Frontiers in Genetics</em> (2017-06-16) <a href="https://doi.org/gcz6m3">https://doi.org/gcz6m3</a><br />
DOI: <a href="https://doi.org/10.3389/fgene.2017.00084">10.3389/fgene.2017.00084</a> · PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28670325">28670325</a> · PMCID: <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5472696">PMC5472696</a></p>
</div>
<div id="ref-lOKGIFIL">
<p>10. <strong>Methods for the integration of multi-omics data: mathematical aspects</strong><br />
Matteo Bersanelli, Ettore Mosca, Daniel Remondini, Enrico Giampieri, Claudia Sala, Gastone Castellani, Luciano Milanesi<br />
<em>BMC Bioinformatics</em> (2016-01-20) <a href="https://doi.org/gcpgct">https://doi.org/gcpgct</a><br />
DOI: <a href="https://doi.org/10.1186/s12859-015-0857-9">10.1186/s12859-015-0857-9</a> · PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/26821531">26821531</a> · PMCID: <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4959355">PMC4959355</a></p>
</div>
<div id="ref-1Hk7MyXmO">
<p>11. <strong>Similarity network fusion for aggregating data types on a genomic scale</strong><br />
Bo Wang, Aziz M Mezlini, Feyyaz Demir, Marc Fiume, Zhuowen Tu, Michael Brudno, Benjamin Haibe-Kains, Anna Goldenberg<br />
<em>Nature Methods</em> (2014-01-26) <a href="https://doi.org/f5v9f5">https://doi.org/f5v9f5</a><br />
DOI: <a href="https://doi.org/10.1038/nmeth.2810">10.1038/nmeth.2810</a> · PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24464287">24464287</a></p>
</div>
<div id="ref-vvARGimo">
<p>12. <a href="https://www.aans.org/Patients/Neurosurgical-Conditions-and-Treatments/Glioblastoma-Multiforme">https://www.aans.org/Patients/Neurosurgical-Conditions-and-Treatments/Glioblastoma-Multiforme</a></p>
</div>
<div id="ref-X7vddlEk">
<p>13. <strong>Synergistic effect of different levels of genomic data for cancer clinical outcome prediction</strong><br />
Dokyoon Kim, Hyunjung Shin, Young Soo Song, Ju Han Kim<br />
<em>Journal of Biomedical Informatics</em> (2012-12) <a href="https://doi.org/f4gstk">https://doi.org/f4gstk</a><br />
DOI: <a href="https://doi.org/10.1016/j.jbi.2012.07.008">10.1016/j.jbi.2012.07.008</a> · PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22910106">22910106</a></p>
</div>
<div id="ref-1rJXR1pc">
<p>14. <strong>A Bayesian Integrative Genomic Model for Pathway Analysis of Complex Traits</strong><br />
Brooke L. Fridley, Steven Lund, Gregory D. Jenkins, Liewei Wang<br />
<em>Genetic Epidemiology</em> (2012-03-28) <a href="https://doi.org/f3xkqr">https://doi.org/f3xkqr</a><br />
DOI: <a href="https://doi.org/10.1002/gepi.21628">10.1002/gepi.21628</a> · PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22460780">22460780</a> · PMCID: <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3894829">PMC3894829</a></p>
</div>
<div id="ref-18IBB9pE2">
<p>15. <strong>Time to Recurrence and Survival in Serous Ovarian Tumors Predicted from Integrated Genomic Profiles</strong><br />
Parminder K. Mankoo, Ronglai Shen, Nikolaus Schultz, Douglas A. Levine, Chris Sander<br />
<em>PLoS ONE</em> (2011-11-03) <a href="https://doi.org/c6zcns">https://doi.org/c6zcns</a><br />
DOI: <a href="https://doi.org/10.1371/journal.pone.0024709">10.1371/journal.pone.0024709</a> · PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22073136">22073136</a> · PMCID: <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3207809">PMC3207809</a></p>
</div>
<div id="ref-DyO707G8">
<p>16. <strong>ATHENA: the analysis tool for heritable and environmental network associations</strong><br />
Emily R. Holzinger, Scott M. Dudek, Alex T. Frase, Sarah A. Pendergrass, Marylyn D. Ritchie<br />
<em>Bioinformatics</em> (2013-10-21) <a href="https://doi.org/gfq96v">https://doi.org/gfq96v</a><br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/btt572">10.1093/bioinformatics/btt572</a> · PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24149050">24149050</a> · PMCID: <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3933870">PMC3933870</a></p>
</div>
<div id="ref-glnTeD5X">
<p>17. <strong>A statistical framework for genomic data fusion</strong><br />
G. R. G. Lanckriet, T. De Bie, N. Cristianini, M. I. Jordan, W. S. Noble<br />
<em>Bioinformatics</em> (2004-05-06) <a href="https://doi.org/bsph7z">https://doi.org/bsph7z</a><br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/bth294">10.1093/bioinformatics/bth294</a> · PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/15130933">15130933</a></p>
</div>
<div id="ref-8kVDErgV">
<p>18. <strong>Protein function prediction via graph kernels</strong><br />
K. M. Borgwardt, C. S. Ong, S. Schonauer, S. V. N. Vishwanathan, A. J. Smola, H.-P. Kriegel<br />
<em>Bioinformatics</em> (2005-06-01) <a href="https://doi.org/c6s8jt">https://doi.org/c6s8jt</a><br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/bti1007">10.1093/bioinformatics/bti1007</a> · PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/15961493">15961493</a></p>
</div>
<div id="ref-14murNoRs">
<p>19. <strong>Fast protein classification with multiple networks</strong><br />
K. Tsuda, H. Shin, B. Scholkopf<br />
<em>Bioinformatics</em> (2005-09-01) <a href="https://doi.org/bhdtqm">https://doi.org/bhdtqm</a><br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/bti1110">10.1093/bioinformatics/bti1110</a> · PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/16204126">16204126</a></p>
</div>
<div id="ref-aPrQxP1f">
<p>20. <strong>Graph sharpening plus graph integration: a synergy that improves protein functional classification</strong><br />
Hyunjung Shin, Andreas Martin Lisewski, Olivier Lichtarge<br />
<em>Bioinformatics</em> (2007-10-31) <a href="https://doi.org/dt399h">https://doi.org/dt399h</a><br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/btm511">10.1093/bioinformatics/btm511</a> · PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/17977886">17977886</a></p>
</div>
<div id="ref-KOD2gdVS">
<p>21. <strong>The properties of high-dimensional data spaces: implications for exploring gene and protein expression data</strong><br />
Robert Clarke, Habtom W. Ressom, Antai Wang, Jianhua Xuan, Minetta C. Liu, Edmund A. Gehan, Yue Wang<br />
<em>Nature Reviews Cancer</em> (2008-01) <a href="https://doi.org/ffksnf">https://doi.org/ffksnf</a><br />
DOI: <a href="https://doi.org/10.1038/nrc2294">10.1038/nrc2294</a> · PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/18097463">18097463</a> · PMCID: <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2238676">PMC2238676</a></p>
</div>
</div>
<script>
// AnchorJS minified version below.
// Source https://github.com/bryanbraun/anchorjs/blob/064abdd0987f305933ec4982af6d0c1cf2fd0814/anchor.js

/**
 * AnchorJS - v4.0.0 - 2017-06-02
 * https://github.com/bryanbraun/anchorjs
 * Copyright (c) 2017 Bryan Braun; Licensed MIT
 */
!function(A,e){"use strict";"function"==typeof define&&define.amd?define([],e):"object"==typeof module&&module.exports?module.exports=e():(A.AnchorJS=e(),A.anchors=new A.AnchorJS)}(this,function(){"use strict";function A(A){function e(A){A.icon=A.hasOwnProperty("icon")?A.icon:"",A.visible=A.hasOwnProperty("visible")?A.visible:"hover",A.placement=A.hasOwnProperty("placement")?A.placement:"right",A.class=A.hasOwnProperty("class")?A.class:"",A.truncate=A.hasOwnProperty("truncate")?Math.floor(A.truncate):64}function t(A){var e;if("string"==typeof A||A instanceof String)e=[].slice.call(document.querySelectorAll(A));else{if(!(Array.isArray(A)||A instanceof NodeList))throw new Error("The selector provided to AnchorJS was invalid.");e=[].slice.call(A)}return e}function n(){if(null===document.head.querySelector("style.anchorjs")){var A,e=document.createElement("style");e.className="anchorjs",e.appendChild(document.createTextNode("")),void 0===(A=document.head.querySelector('[rel="stylesheet"], style'))?document.head.appendChild(e):document.head.insertBefore(e,A),e.sheet.insertRule(" .anchorjs-link {   opacity: 0;   text-decoration: none;   -webkit-font-smoothing: antialiased;   -moz-osx-font-smoothing: grayscale; }",e.sheet.cssRules.length),e.sheet.insertRule(" *:hover > .anchorjs-link, .anchorjs-link:focus  {   opacity: 1; }",e.sheet.cssRules.length),e.sheet.insertRule(" [data-anchorjs-icon]::after {   content: attr(data-anchorjs-icon); }",e.sheet.cssRules.length),e.sheet.insertRule(' @font-face {   font-family: "anchorjs-icons";   src: url(data:n/a;base64,AAEAAAALAIAAAwAwT1MvMg8yG2cAAAE4AAAAYGNtYXDp3gC3AAABpAAAAExnYXNwAAAAEAAAA9wAAAAIZ2x5ZlQCcfwAAAH4AAABCGhlYWQHFvHyAAAAvAAAADZoaGVhBnACFwAAAPQAAAAkaG10eASAADEAAAGYAAAADGxvY2EACACEAAAB8AAAAAhtYXhwAAYAVwAAARgAAAAgbmFtZQGOH9cAAAMAAAAAunBvc3QAAwAAAAADvAAAACAAAQAAAAEAAHzE2p9fDzz1AAkEAAAAAADRecUWAAAAANQA6R8AAAAAAoACwAAAAAgAAgAAAAAAAAABAAADwP/AAAACgAAA/9MCrQABAAAAAAAAAAAAAAAAAAAAAwABAAAAAwBVAAIAAAAAAAIAAAAAAAAAAAAAAAAAAAAAAAMCQAGQAAUAAAKZAswAAACPApkCzAAAAesAMwEJAAAAAAAAAAAAAAAAAAAAARAAAAAAAAAAAAAAAAAAAAAAQAAg//0DwP/AAEADwABAAAAAAQAAAAAAAAAAAAAAIAAAAAAAAAIAAAACgAAxAAAAAwAAAAMAAAAcAAEAAwAAABwAAwABAAAAHAAEADAAAAAIAAgAAgAAACDpy//9//8AAAAg6cv//f///+EWNwADAAEAAAAAAAAAAAAAAAAACACEAAEAAAAAAAAAAAAAAAAxAAACAAQARAKAAsAAKwBUAAABIiYnJjQ3NzY2MzIWFxYUBwcGIicmNDc3NjQnJiYjIgYHBwYUFxYUBwYGIwciJicmNDc3NjIXFhQHBwYUFxYWMzI2Nzc2NCcmNDc2MhcWFAcHBgYjARQGDAUtLXoWOR8fORYtLTgKGwoKCjgaGg0gEhIgDXoaGgkJBQwHdR85Fi0tOAobCgoKOBoaDSASEiANehoaCQkKGwotLXoWOR8BMwUFLYEuehYXFxYugC44CQkKGwo4GkoaDQ0NDXoaShoKGwoFBe8XFi6ALjgJCQobCjgaShoNDQ0NehpKGgobCgoKLYEuehYXAAAADACWAAEAAAAAAAEACAAAAAEAAAAAAAIAAwAIAAEAAAAAAAMACAAAAAEAAAAAAAQACAAAAAEAAAAAAAUAAQALAAEAAAAAAAYACAAAAAMAAQQJAAEAEAAMAAMAAQQJAAIABgAcAAMAAQQJAAMAEAAMAAMAAQQJAAQAEAAMAAMAAQQJAAUAAgAiAAMAAQQJAAYAEAAMYW5jaG9yanM0MDBAAGEAbgBjAGgAbwByAGoAcwA0ADAAMABAAAAAAwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABAAH//wAP) format("truetype"); }',e.sheet.cssRules.length)}}this.options=A||{},this.elements=[],e(this.options),this.isTouchDevice=function(){return!!("ontouchstart"in window||window.DocumentTouch&&document instanceof DocumentTouch)},this.add=function(A){var i,o,s,c,r,a,h,l,u,d,f,g,p=[];if(e(this.options),"touch"===(g=this.options.visible)&&(g=this.isTouchDevice()?"always":"hover"),A||(A="h2, h3, h4, h5, h6"),0===(i=t(A)).length)return this;for(n(),o=document.querySelectorAll("[id]"),s=[].map.call(o,function(A){return A.id}),r=0;r<i.length;r++)if(this.hasAnchorJSLink(i[r]))p.push(r);else{if(i[r].hasAttribute("id"))c=i[r].getAttribute("id");else if(i[r].hasAttribute("data-anchor-id"))c=i[r].getAttribute("data-anchor-id");else{u=l=this.urlify(i[r].textContent),h=0;do{void 0!==a&&(u=l+"-"+h),a=s.indexOf(u),h+=1}while(-1!==a);a=void 0,s.push(u),i[r].setAttribute("id",u),c=u}d=c.replace(/-/g," "),(f=document.createElement("a")).className="anchorjs-link "+this.options.class,f.href="#"+c,f.setAttribute("aria-label","Anchor link for: "+d),f.setAttribute("data-anchorjs-icon",this.options.icon),"always"===g&&(f.style.opacity="1"),""===this.options.icon&&(f.style.font="1em/1 anchorjs-icons","left"===this.options.placement&&(f.style.lineHeight="inherit")),"left"===this.options.placement?(f.style.position="absolute",f.style.marginLeft="-1em",f.style.paddingRight="0.5em",i[r].insertBefore(f,i[r].firstChild)):(f.style.paddingLeft="0.375em",i[r].appendChild(f))}for(r=0;r<p.length;r++)i.splice(p[r]-r,1);return this.elements=this.elements.concat(i),this},this.remove=function(A){for(var e,n,i=t(A),o=0;o<i.length;o++)(n=i[o].querySelector(".anchorjs-link"))&&(-1!==(e=this.elements.indexOf(i[o]))&&this.elements.splice(e,1),i[o].removeChild(n));return this},this.removeAll=function(){this.remove(this.elements)},this.urlify=function(A){var t=/[& +$,:;=?@"#{}|^~[`%!'<>\]\.\/\(\)\*\\]/g;return this.options.truncate||e(this.options),A.trim().replace(/\'/gi,"").replace(t,"-").replace(/-{2,}/g,"-").substring(0,this.options.truncate).replace(/^-+|-+$/gm,"").toLowerCase()},this.hasAnchorJSLink=function(A){var e=A.firstChild&&(" "+A.firstChild.className+" ").indexOf(" anchorjs-link ")>-1,t=A.lastChild&&(" "+A.lastChild.className+" ").indexOf(" anchorjs-link ")>-1;return e||t||!1}}return A});

// Enable links for selected headers
var anchors = new AnchorJS();
anchors.add("h2, h3, h4")
</script>
<!-- Enable Hypothesis annotation. https://web.hypothes.is/ -->
<script src="https://hypothes.is/embed.js" async></script>
</body>
</html>
